Makefile

```diff
@@ -0,0 +1,43 @@
+SHELL := /bin/bash
+
+.PHONY: help prepare-demo
+.DEFAULT_GOAL := help
+
+help: ## 💬 This help message :)
+	@grep -E '[a-zA-Z_-]+:.*?## .*$$' $(firstword $(MAKEFILE_LIST)) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+lint: ## 🔍 Lint the code base (but don't fix)
+	@echo -e "\e[34m$@\e[0m" || true
+	@./lint.sh
+
+lint-fix: ## 🌟 Lint and fix the code base
+	@echo -e "\e[34m$@\e[0m" || true
+	@./lint.sh -f
+
+build: ## 🔨 Build the Banking Application
```

Comment:
What do you think is the best way to make changes when we add the second sample?

I can think of the following directory structure for example:
```
.
|-- Makefile # Common commands like `lint`. Also, we can add commands to do something for all app like `test-all`.
|-- banking-app
|    |-- Makefile # `test`, `prepare-demo`, etc for banking-app
|--second-app
    |--Makefile # `test`, `prepare-demo`, etc for second-app

```

Or just we can edit this Makefile and provide command like `make test banking-app` ?

Reply:
I have added another makefile in the banking-app folder. If you run `make build` from the root you will get the message "Please change directory to the sample you wish to build." When we get a second application I think we can improve this, but for now, you have to be inside the banking-app folder to run `make build` etc.